[2024-09-25T11:06:03.989+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T11:06:04.032+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.choose_modes manual__2024-09-25T11:05:57.968197+00:00 [queued]>
[2024-09-25T11:06:04.054+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.choose_modes manual__2024-09-25T11:05:57.968197+00:00 [queued]>
[2024-09-25T11:06:04.055+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T11:06:04.110+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): choose_modes> on 2024-09-25 11:05:57.968197+00:00
[2024-09-25T11:06:04.128+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=471) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T11:06:04.131+0000] {standard_task_runner.py:72} INFO - Started process 485 to run task
[2024-09-25T11:06:04.135+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'choose_modes', 'manual__2024-09-25T11:05:57.968197+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpge2vof31']
[2024-09-25T11:06:04.142+0000] {standard_task_runner.py:105} INFO - Job 136: Subtask choose_modes
[2024-09-25T11:06:04.288+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.choose_modes manual__2024-09-25T11:05:57.968197+00:00 [running]> on host e10075799413
[2024-09-25T11:06:04.571+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='choose_modes' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T11:05:57.968197+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T11:05:57.968197+00:00'
[2024-09-25T11:06:04.573+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T11:06:04.580+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-09-25T11:06:04.636+0000] {python.py:240} INFO - Done. Returned value was: init
[2024-09-25T11:06:04.637+0000] {branch.py:38} INFO - Branch into init
[2024-09-25T11:06:04.639+0000] {branch.py:64} INFO - Expanding task group init into ['init.create_bucket']
[2024-09-25T11:06:04.641+0000] {skipmixin.py:230} INFO - Following branch ('init.create_bucket',)
[2024-09-25T11:06:04.693+0000] {skipmixin.py:278} INFO - Skipping tasks [('pipeline.staging.dellstore_db.extract.customers', -1), ('pipeline.staging.dellstore_db.extract.products', -1), ('pipeline.staging.dellstore_db.extract.inventory', -1), ('pipeline.staging.dellstore_db.extract.orders', -1), ('pipeline.staging.dellstore_db.extract.cust_hist', -1), ('pipeline.staging.dellstore_db.extract.categories', -1), ('pipeline.staging.dellstore_db.extract.orderlines', -1)]
[2024-09-25T11:06:04.800+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T11:06:04.801+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=choose_modes, run_id=manual__2024-09-25T11:05:57.968197+00:00, execution_date=20240925T110557, start_date=20240925T110604, end_date=20240925T110604
[2024-09-25T11:06:04.900+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T11:06:05.129+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T11:06:05.134+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
